index.html

```diff
@@ -303,7 +300,7 @@ <h1>
     <section class="single-page scrollblock" id="screenshots">
         <div class="">
             <div class="align">
-                <img alt="download-icon" style="size:10%" src="image/screenshot.png">
```

Comment:
can i get the image of screenshots whether they are looking same or not ?
If yes great work finding this and removing it

Reply:
Size is no longer supported I removed a it and it works.

Reply:
Size is not supported now

Reply:
Is it? I guess, I am not aware of it, it looks good to me just I am a bit paranoid for building something that will break the page